### DISCUSSION TITLE
Any chance gettings some help from Aqara?

### DISCUSSION THREAD

**Discussion by lindhe** [zigpy/zha-device-handlers#3403] [Q&A]
Considering that [Aqara recently joined the "Works with Home Assistant" program](https://www.home-assistant.io/blog/2024/09/03/aqara-joins-works-with-home-assistant/), do you think it would be possible to get some eyes and hands from them in this repo? There's plenty of open issues regarding Aqara devices, and if we can get some help from the vendor that would be great.

Any idea how we can approach this? I have no insights into the "Works with Home Assistant" program, but I hope it's more than just a meaningless promise from them.

**Comment by Hedda** [comment]
+1 though the current "Works with Home Assistant" partnership rules state that manufacturers are exempt from the duty of supporting and maintaining of Home Assistant's built-in native Zigbee gateway integration (i.e. the [ZHA integration](https://www.home-assistant.io/integrations/zha)) I would imagine that it should be in Aqara's best interest to make their all their newer Zigbee devices work out-of-the-box via the [ZHA integration](https://www.home-assistant.io/integrations/zha) with full feature-parity compared to when the same Zigbee device is used via their own proprietary Zigbee gateways (e.g. their official Aqara Hub series). 

* https://partner.home-assistant.io

Still, could be great if Aqara begins to actively contribute to ZHA Device Handlers, the zha library, and/or the ZHA integration component, 

* https://www.home-assistant.io/integrations/zha#how-to-add-support-for-new-and-unsupported-devices

And even more awesome if they would also start to provide Zigbee OTA (Over-The-Air) firmware image and/or OTAU provider(s) for ZHA:

* https://github.com/zigpy/zigpy/wiki/OTA-Information-for-Manufacturers

Aqara already offers a huge collection of native Zigbee devices so it would be very cool if could be said that Aqara officially supported ZHA.

* https://zigbee.blakadder.com/vendors.html

> _Considering that [Aqara recently joined the "Works with Home Assistant" program](https://www.home-assistant.io/blog/2024/09/03/aqara-joins-works-with-home-assistant/), do you think it would be possible to get some eyes and hands from them in this repo? There's plenty of open issues regarding Aqara devices, and if we can get some help from the vendor that would be great._

Noted that their announcement does not actually specifically mention the ZHA integration, but instead only focusing on ther support for Matter, however it does indirectly mention Zigbee devices but only in reference to them being used via their Matter Bridge, (so not ZHA).

> _Any idea how we can approach this? I have no insights into the "Works with Home Assistant" program, but I hope it's more than just a meaningless promise from them._

Maybe raise the question via forum discussion -> https://community.home-assistant.io/t/aqara-joins-works-with-home-assistant/766931/

I believe that the general idea with the "Works with Home Assistant" program is that if a manufacturer partnering with the Open Home Foundation also wants to use the "**_Works via Zigbee with Home Assistant_**" badge in the future for marketing devices then they are directly responsible to somehow maintaining the integrity of how their own devices work with the ZHA integration. See:

Note that the "Works with Home Assistant" program's guidelines specification were recently changed from getting an approval per brand to require that each and every device needs to be certified by Nabu Casa separately if the manufacturer wants to use the "**_Works via Zigbee with Home Assistant_**" badge to market that device:

* https://www.home-assistant.io/blog/2024/08/08/works-with-home-assistant-becomes-part-ohf/

As such I would expect that even if Aqara decide to to use those badges then we can probably expect them to only focus on newest and/or upcoming devices that they still manufacture and sell for them to get return on their investments (at least if their marketing team have anything to say about it): 

* https://www.aqara.com/us/product/
